app/views/new_document/choose_document_type.html.erb

```diff
@@ -18,7 +18,7 @@
               gtm: "choose-document-type",
               "gtm-value": document_type.label,
             },
-            conditional: document_type.hint_govspeak ? tag.div(govspeak_to_html(document_type.hint_govspeak), class: "govuk-body") : nil,
+            conditional: document_type.hint_govspeak ? render_govspeak(document_type.hint_govspeak) : nil,
```

Comment:
There are a few other cases where we call `govspeak_to_html` in the views, we should probably replace those calls with `render_govspeak` too?

That would then suggest we could remove the `govspeak_to_html` method altogether but I think it would still be needed by the tests 🤔

Reply:
Agree. I think we're currently using `govspeak_to_html` in `render_govspeak` – have to double-check. I guess the question is shall we keep it and make it private or move the `govspeak_to_html` logic straight into `render_govspeak`?

Reply:
Yes, ideally we could move the `govspeak_to_html` logic into `render_govspeak` and delete the method. Not sure if swapping `govspeak_to_html` for `render_govspeak` inside the tests will break things (the additional dependency / runtime complexities associated with the `render` call). Worth a try!

Reply:
Did that refactor in a separate commit and tested visually to make sure it renders correctly.